r/scenebuilder

```diff
@@ -0,0 +1 @@
+/nix/store/230ga7fzgvdfddd38bgpi4xgyv8w7y4w-scenebuilder-15.0.0
```

Comment:
Does files are not supposed to be commited.

Reply:
m( damn, fixed.